papers/doc/7-systems/x1-010.md

```diff
@@ -31,8 +31,8 @@ In furnace, You can set envelope shape split mode. When it sets, its waveform wi
 - `20xx`: set PCM frequency (1 to FF).
 - `22xx`: set envelope mode.
   - bit 0 sets whether envelope will affect this channel.
-  - bit 1 sets whether envelope one-shot mode. when it sets, channel is halted after envelope cycle is ended.
-  - bit 2 sets whether envelope shape split mode. when it sets, envelope shape will splitted to left half and right half.
+  - bit 1 toggles the envelope one-shot mode. when it is set, channel is halted after envelope cycle is finished.
```

Comment:
Note that this change may also require updates to ay8910.md, ay8930.md and ym2610.md as well.